README.md

```diff
@@ -31,40 +34,21 @@ make menuconfig
 # Customize the linux config, if desired
 make linux-menuconfig
 
-# Build the firmware
-make
+# Build using Buildroots official Docker image
+sudo docker run -v $(dirname $(pwd)):$(dirname $(pwd)) buildroot/base bash -c "cd $(pwd); make BR2_JLEVEL=$(nproc)"
+
+# Or build the firmware locally. See buildroot docs for dependencies
+make BR2_JLEVEL=$(nproc)
 ```
 
-The firmware image will be built in
+The firmware image will be built as
 `images/emmc.img`.
 
 Use separate build directories to simplify working with multiple
-configurations.  The builds will not interfere with each other and
+configurations. The builds will not interfere with each other and
 switching between configurations won't require a clean and full
 rebuild.
 
-``` bash
-cd build
-
-# Initialize a build directory for one config
-mkdir sama5d2_xplained_xaprc
-pushd sama5d2_xplained_xaprc
-
-make O=$PWD -C ../.. sama5d2_xplained_xaprc_defconfig
```

Comment:
Can you keep the example code, up to this line?  I refer back to it for the correct syntax for initializing the defconfig from a separate build directory.  (The `make -O=$PWD -C ../.. <defconfig>` syntax).